api/envoy/config/cluster/v3/outlier_detection.proto

```diff
@@ -151,7 +151,8 @@ message OutlierDetection {
   google.protobuf.UInt32Value failure_percentage_request_volume = 20;
 
   // The maximum time that a host is ejected for. See :ref:`base_ejection_time<envoy_api_field_config.cluster.v3.OutlierDetection.base_ejection_time>`
-  // for more information.
+  // for more information. If not specified, the default value or
+  // :ref:`base_ejection_time<envoy_api_field_config.cluster.v3.OutlierDetection.base_ejection_time>` value is applied, whatever is larger.
   // Defaults to 300000ms or 300s.
```

Comment:
I would inline the default here instead of referring to the default on L154 and defining it on L156

Reply:
+1